articles/security/bulletins/cve-2017-16897.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Auth0 Security Bulletin CVE 2017-16897
+description: Security Update for passport-wsfed-saml2 Passport strategy library (CVE 2017-16897)
+toc: true
+---
+# Security Update for passport-wsfed-saml2 Passport Strategy Library
+
+**Published**: December 22, 2017
```

Comment:
@goduncan Is this valid? I'm not sure if this date is meant to be when this doc will be published or something else (?)

Reply:
Yes this is the date we publish the doc. So if we merge today it's correct.